src/elements/content-sidebar/activity-feed/comment/__tests__/Replies.test.js

```diff
@@ -170,6 +172,27 @@ describe('elements/content-sidebar/ActivityFeed/comment/Replies', () => {
         expect(replyCreate).toBeCalledWith('Batman');
     });
 
+    test('should call onReplyDelete when reply is deleted', () => {
+        getWrapper({
+            replies: [comment, comment2],
+        });
+
+        const [menu] = screen.getAllByTestId('comment-actions-menu');
```

Comment:
Can you help me understand why `getAll-` here instead of just `get-`?

Reply:
Also, lets `getByLabel("Options")` vis-a-vis https://testing-library.com/docs/queries/about/#priority We get some free a11y verification that way :)